Sources/MediaScrubProgressBar.swift

```diff
@@ -0,0 +1,91 @@
+/*****************************************************************************
+ * MediaScrubProgressBar.swift
+ *
+ * Copyright © 2019 VLC authors and VideoLAN
+ *
+ * Authors: Robert Gordon <robwaynegordon@gmail.com>
+ *
+ *
+ * Refer to the COPYING file of the official project for license.
+ *****************************************************************************/
+
+@objc (VLCMediaProgressBarDelegate)
+protocol MediaScrubProgressBarDelegate {
+    func didMoveMediaScrubProgressSlider(_ progressBar: MediaScrubProgressBar, sender: UISlider)
+}
+
+@objc (VLCMediaProgressBar)
+@objcMembers class MediaScrubProgressBar: UIView {
+    
+    // MARK: Instance Variables
+    weak var delegate: MediaScrubProgressBarDelegate?
+    
+    lazy var progressSlider: UISlider = {
+        var slider = UISlider()
+        slider.minimumValue = 0
+        slider.maximumValue = 100
+        slider.minimumTrackTintColor = .orange
+        slider.maximumTrackTintColor = UIColor(red: 1, green: 1, blue: 1, alpha: 0.2)
+        slider.setThumbImage(UIImage(named: "sliderThumb"), for: .normal)
+        slider.thumbTintColor = .orange
+        slider.isContinuous = true
+        slider.addTarget(self, action: #selector(moveSliderThumb), for: .touchUpInside)
+        slider.translatesAutoresizingMaskIntoConstraints = false
+        return slider
+    }()
+    
+    var elapsedTimeLabel: UILabel = {
+        var label = UILabel()
+        label.font = UIFont(name: "SFProDisplay-Bold", size: 12)
+        label.textColor = .orange
+        label.text = "000:00"
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    var remainingTimeLabel: UILabel = {
+        var label = UILabel()
+        label.font = UIFont(name: "SFProDisplay-Bold", size: 12)
```

Comment:
don't use this font directly !! Systemfont is the way to go. if apple changes the standard font this breaks the UI ^